quickstart/deploy-mayastor.md

```diff
@@ -6,7 +6,7 @@
 Before deploying and using Mayastor please consult the [Known Issues](https://mayastor.gitbook.io/introduction/quickstart/known-issues) section of this guide.
 {% endhint %}
 
-The steps and commands which follow are intended only for use in conjunction with Mayastor version(s) 2.1.x.
+The steps and commands which follow are intended only for use in conjunction with Mayastor version(s) 2.3.x.
```

Comment:
The version here acts more as the base since when the steps are valid. We can modify as "2.1.x and above".